test/compute_jacobian_eigen.jl

```diff
@@ -0,0 +1,134 @@
+using Test
+using DFTK
+using LinearMaps
+import DFTK: ortho_qr, pack_ψ, unpack_ψ, proj_tangent, proj_tangent!
+import DFTK: apply_K, apply_Ω, filled_occupation
+import DFTK: precondprep!, FunctionPreconditioner
+
+include("testcases.jl")
+
+@testset "Compute eigenvalues" begin
+    if mpi_nprocs() == 1  # Distributed implementation not yet available
```

Comment:
Better put this around the testset. Then it does not even show up in the MPI output.